types/node-sass/index.d.ts

```diff
@@ -1,56 +1,398 @@
-// Type definitions for Node Sass v3.10.1
+// Type definitions for node-sass 4.11.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
If re-enabling too many rules is cumbersome, as many as is reasonable would go a long way.

Reply:
@DanielRosenwasser Thanks for reviewing! I've sorted out the build and re-enabled all the linters except for one. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33017/commits/ad1f840706fde09ed2da1ef83f19111b22381223